maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -1560,6 +1566,18 @@ private void convertGroupParameters()
         }
     }
 
+    private void convertJunitEngineParameters()
+    {
+        if ( this.getJunitIncludeEngine() != null )
```

Comment:
Pls use `getJunitIncludeEngine()` instead of `this.getJunitIncludeEngine()`.
Remove "this".

Reply:
Ok, it was done previously by IDE formatting but I will change this :)